sklearn/tests/test_docstring_parameters.py

```diff
@@ -145,6 +146,12 @@ def test_tabs():
 
         # because we don't import
         mod = importlib.import_module(modname)
+
+        # unwrap to get module because Pep562 backport wraps the original
+        # module
```

Comment:
Mark as to be removed in 0.24 or when Pep562 isn't used anymore

?

Reply:
We can also used this for experimental features for a better error message?

Reply:
still worth a tag?